### ISSUE TITLE
Page scrolls to top after initial load.

### ISSUE THREAD

**Issue by davisagli** [plone/volto#6518]
If I start to load a page (initial load via SSR) and start to scroll down, the page scrolls back to the top when it is done loading, which interrupts me. I guess this comes from the ScrollToTop component connected to the router. This is okay for navigating after the initial load (with some exceptions like #3662) but we should try to avoid it for the initial load.

**Comment by CoyoteClear340** [comment]
I'll be taking on this issue and will make sure to keep you updated on my progress. Thank you for your patience and understanding as I work through it

**Comment by fredvd** [comment]
We have a different scrolltotop issue, at the moment in Volto 18.1.1 with searchblocks, where the opposite happens. If you click on an item halfway the search results, the page navigates to the item, but the page is never scrolled to the top.  

@davisagli do you think this is releated as well to the ScrollToTop component, or does it warrant its own issue?

**Comment by davisagli** [comment]
@fredvd I don't know whether that is related to ScrollToTop or not, but it sounds like it should be a separate issue, with steps to reproduce it, please.
